docs/future_plans.md

```diff
@@ -0,0 +1,62 @@
+**Last updated:** January 2021.
+
+**Tl;dr:** We'll be redesigning how Stardoc works, and deprioritizing feature
+requests and minor bugs until that work is complete (targeting 2021 Q2).
+
+## Technical motivation
+
+Stardoc is currently the recommended tool for generating documentation of
+Starlark rules. It [replaces](skydoc_deprecation.md) Skydoc, the previous tool,
+which worked by mocking evaluation of .bzl files as Python code.
```

Comment:
Perhaps: "which worked by evaluating .bzl files in a Python interpreter, using fake versions of the functions of Bazel's build language".

Reply:
Done. Capitalized "Build Language" though -- that's the convention we agreed on, right?

Reply:
Yep, thanks.